frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -166,6 +167,18 @@ public EnhancedGraphTable(Project project) {
     subscribeToMacheteFileChange();
   }
 
+  public void refreshMacheteFile() {
```

Comment:
It's a little hack for the UITest to work properly, basicly the UITest now modifies the machete file not using the VFS API. Because of that we cannot ensure when the change will be seen by VFS, so we tell the VFS to refresh machete file explicitly during the test. This does VFS update delay is not noticable when working with the plugin as tested with `runIde`, but the test needs to have to have consistnet results.

Reply:
Actually, pls put this comment of yours ☝🏻 directly into code (either here or in UI test code), so that it's not lost